scripts/create_release.py

```diff
@@ -50,7 +53,12 @@ def main(args):
     f.write(json.dumps(release_info, indent=2))
     f.write('\n')
 
-  subprocess.check_call(os.path.join(script_dir, 'update_bazel_workspace.sh'), cwd=root_dir)
+  subprocess.check_call([os.path.join(script_dir, 'update_bazel_workspace.sh')], cwd=root_dir)
+
+  branch_name = 'test_version_%s' % '_'.join(str(part) for part in new_version)
```

Comment:
do you want to take the 'test_' prefix back out again?

Reply:
oops.. thanks.